meta/recipes-core/udev/eudev_3.2.bb

```diff
@@ -54,6 +56,10 @@ do_install_append() {
 
 	# hid2hci has moved to bluez4. removed in udev as of version 169
 	rm -f ${D}${base_libdir}/udev/hid2hci
+
+	# mock an older udev version
+	install -d ${D}${libdir}
+	ln -rs ${D}${base_libdir}/libudev.so.1 ${D}${libdir}/libudev.so.0
```

Comment:
What does `-r` do? I can't find it in https://busybox.net/downloads/BusyBox.html nor in the GNU `ln` man page

Reply:
It does come up in my GNU `ln` man page:

```
-r, --relative
              create symbolic links relative to link location
```

This apparently is the normal way to do it instead of linking to an absolute path. I can't give you a good reason for this other than, the `the link will also be correct while still on the build system`.